vispy/color/colormap.py

```diff
@@ -986,7 +1046,7 @@ def __init__(self, limits=(0.33, 0.66, 1.0)):
     winter=_Winter(),
     light_blues=_SingleHue(),
     orange=_SingleHue(hue=35),
-    viridis=Colormap(ColorArray(_viridis_data[::4])),
+    viridis=Colormap(ColorArray(_viridis_data[::1])),
```

Comment:
ahh never mind, I see you have. This can just be `(_viridis_data)`, then, no need to slice.